Quantum-Control-Applications/NV-Centers/Confocal-Setup/power_rabi.py

```diff
@@ -0,0 +1,81 @@
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm.qua import *
+import matplotlib.pyplot as plt
+from configuration import *
+
+###################
+# The QUA program #
+###################
+
+a_min = 0.1  # proportional factor to the pulse amplitude
+a_max = 1  # proportional factor to the pulse amplitude
+da = 0.02
+a_vec = np.arange(a_min, a_max + da / 2, da)  # +da/2 to include a_max
```

Comment:
maybe say that a_min < d/2?

Reply:
Not sure what you mean here?

Reply:
a_min = 0.9
da = 0.1
a_max = 1

I get, 0.9, 1.0

Don't see the issue...
The problem is if you choose a_min = 0.91.
In this case you'll also get 1.1, which is higher than a_max.
However, it will also be included in the scan, so it is ok...

The only real issue is if you pick a_min=0.95, and then 1.05 will not be included in numpy `arange`, but **might** be included in our scan.

Reply:
you're right, the relation is actually more annoying, if you choose 0< a_min< d/2 +n*da . But it is still there.
for instance
a_min=0.02/0.12/0.22...
da=0.1
a_max=1
then you also get 1.02

Reply:
Yes, but this is the same as you would get using numpy `arange`.
With this I am at peace...

Reply:
I mean that numpy `arange` would give the same results are our `for_` loop, so I am ok with that.

Reply:
I just noticed you change the for_ loop condition for a fixed variable, My bad, ignore